src/main/java/domain/ladder/Height.java

```diff
@@ -1,19 +1,24 @@
-package domain;
+package domain.ladder;
 
 import java.util.Objects;
 
 public class Height {
 
-    public static final int MIN_OF_HEIGHT = 1;
+    public static final int MIN_OF_HEIGHT = 0;
     public static final int MAX_OF_HEIGHT = 100;
 
-    private final int height;
+    private int height;
 
     public Height(int height) {
         validate(height);
         this.height = height;
     }
 
+    public void moveUp() {
+        validate(height + 1);
+        this.height += 1;
+    }
```

Comment:
원시값 랩핑은 불변의 속성도 가져가면 더욱 좋은데요, 
불변은 코딩하기 불편하게 만드는 제약이지만 그 제약이 가져다주는 장점들이 있습니다 (불변의 효능은 인터넷에 많으니 심심하시면 찾아보세요)

final을 유지하면서 moveUp 메서드를 유지하는 방법은 다음과 같습니다
```suggestion
    public Height moveUp() {
        return new Height(height + 1);
    }
```

Reply:
해당 코드를 작성할 때 원시값 랩핑이기에 불변 속성을 유지하는 것이 좋을 것 같은데